Invisible recording transfers and Core camera coordinate system clarification (#407)
pupil-labs/pupil-docs: src/invisible/user-guide/invisible-companion-app.md

Pupil Monitor will warn you if a device is not in sync.

More info on the technical implementation and quality of NTP synchronisation can be found in the [Pupil Invisible developer docs](/developer/invisible/#time-synchronization "Pupil Invisible developer docs - time synchronization").

## Recording transfer
### Pupil Cloud

Invisible recordings are uploaded to the cloud from the Invisible Companion App to ensure stability, robustness and security. If you still have the recordings on the Companion device, follow these instructions:

1) Enable 'Cloud Upload' in the Companion App settings
2) Wait until your recording/s are uploaded to Pupil Cloud. In the recording view, uploaded recordings will show a cloud symbol with a tick
3) Login to Pupil Cloud to view or download your recording/s!

### Local transfer
1) Connect your OnePlus device to a PC via USB (using the USB cable supplied)

> 2) Slide down from the top of the device's home-screen and click on 'Android System.USB charging this device'

Reviewer: ```suggestion
2) Slide down from the top of the device's home-screen and click on 'Android System - USB charging this device'
```

3) Click on 'Tap for more options'
4) Select 'Transfer Files'
5) Open File Browser on your PC and access the Internal shared storage of your OnePlus device

> 6) Pupil Invisible recordings can be found in the Pupil Invisible folder

Reviewer: ```suggestion
6) Pupil Invisible recordings can be found in the Pupil Invisible folder

::: tip
On macOS, you require the <a href="https://www.android.com/filetransfer/" alt="Android File Transfer website">Android File Transfer</a> to browse and transfer files between your Mac computer and your Android device.
:::
```

Author: Thanks for that mac info!

Author: Thanks for the mac info!

Pupil Monitor will warn you if a device is not in sync.

More info on the technical implementation and quality of NTP synchronisation can be found in the [Pupil Invisible developer docs](/developer/invisible/#time-synchronization "Pupil Invisible developer docs - time synchronization").

## Recording transfer
### Pupil Cloud

> Invisible recordings are uploaded to the cloud from the Invisible Companion App to ensure stability, robustness and security. If you still have the recordings on the Companion device, follow these instructions:

Reviewer 2: ```suggestion
Pupil Invisible recordings can be uploaded cloud storage directly from the Invisible Companion App. We recommend using Pupil Cloud for ease of user experience, stability, robustness, and data security. 

If you made recordings on Pupil Invisible Companion while cloud upload feature was disabled, you can re-enable and upload recordings to Cloud:```

1) Enable 'Cloud Upload' in the Companion App settings
2) Wait until your recording/s are uploaded to Pupil Cloud. In the recording view, uploaded recordings will show a cloud symbol with a tick

> 3) Login to Pupil Cloud to view or download your recording/s!

Reviewer 2: ```suggestion
3) Login to [Pupil Cloud](https://cloud.pupil-labs.com "Pupil Cloud") to playback recordings, enrich data, and download.
```
